lib/rails_graylog/logger.rb

```diff
@@ -38,7 +38,7 @@ def debug(message = nil, &_block)
     end
 
     def log(severity, message = nil, _progname = nil, &_block)
-      return if message.nil? || message.empty?
+      return if message.nil? || message.empty? || SEV_LABEL.index(severity) < SEV_LABEL.index(level)
```

Comment:
use @level as level is unavailable here:

```ruby
SEV_LABEL.index(severity) < SEV_LABEL.index(@level)
```

Reply:
it was available through attr_accessor so no need to use `@`